__tests__/constants.json

```diff
@@ -1,6 +1,7 @@
 {
   "imageSnapshotOptions": {
-    "failureThreshold": 10,
-    "failureThresholdType": "pixel"
+    "customDiffConfig": {
+      "threshold": 0.15
```

Comment:
so is this percentage now? Just checking

Reply:
`0.14` is the minimal, will go with it.